api/applib/src/main/java/org/apache/causeway/applib/services/publishing/spi/EntityPropertyChangeSubscriber.java

```diff
@@ -48,4 +49,6 @@ public interface EntityPropertyChangeSubscriber extends HasEnabling {
      */
     void onChanging(EntityPropertyChange entityPropertyChange);
 
+    default void onBulkChanging(Can<EntityPropertyChange> entityPropertyChanges) {
```

Comment:
Shouldn't the default implementation iterate over the `Can<>` and call `onChanging(...)` ?

Reply:
Makes sense

Reply:
Fixed

Reply:
Fixed